Framework/Algorithms/src/GeneratePythonScript.cpp

```diff
@@ -133,6 +133,7 @@ void GeneratePythonScript::exec() {
       "#Python Script Generated by GeneratePythonScript Algorithm\n";
   generatedScript += "#########################################################"
                      "#############\n";
+  generatedScript += "from mantid.simpleapi import *\n";
```

Comment:
Would it be better to add an extra `\n` so that it has a proper separation between the import and the code.

Reply:
Will do